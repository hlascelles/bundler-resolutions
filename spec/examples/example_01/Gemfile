# frozen_string_literal: true
# This example will produce lock file with a thor version of 1.3.1, as it is brought in by figjam.
# thor however will not appear in the `DEPENDENCIES` section, as we are not saying it is a dependency of our project.
source "https://rubygems.org"

plugin "bundler-resolutions", path: "../../../"

gem "dememoize"
gem "figjam"

group :resolutions do
  gem "thor", "1.3.1"
end
