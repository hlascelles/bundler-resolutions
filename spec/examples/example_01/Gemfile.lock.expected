GEM
  remote: https://rubygems.org/
  specs:
    dememoize (0.1.1)
    figjam (1.6.2)
      thor (>= 0.14.0, < 2)
    thor (1.3.1)

PLATFORMS
  ruby
  x86_64-linux

DEPENDENCIES
  dememoize
  figjam

BUNDLED WITH
   2.5.14
